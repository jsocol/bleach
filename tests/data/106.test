<div id="36"><a style="behavior:url(#default#AnchorClick);" folder="javascript:alert(36)">XXX</a>//["'`-->]]>]</div>
