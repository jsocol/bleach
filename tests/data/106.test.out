<div id="36"><a style="behavior:url(#default#AnchorClick);">XXX</a>//["'`--&gt;]]&gt;]</div>
